ucm2/AMD/acp3xalc5682m98/HiFi/1mic.conf

```diff
@@ -0,0 +1,89 @@
+SectionVerb {
+	EnableSequence [
+		cdev "hw:${CardId}"
+		cset "name='Headphone Jack Switch' off"
+		cset "name='HPOL Playback Switch' off"
+		cset "name='HPOR Playback Switch' off"
+		cset "name='Stereo1 DAC MIXL DAC L1 Switch' 0"
+		cset "name='Stereo1 DAC MIXR DAC R1 Switch' 0"
+		cset "name='Stereo1 ADC L Mux' 0"
+		cset "name='STO1 ADC Capture Switch' on"
+		cset "name='RECMIX1L CBJ Switch' 1"
+		cset "name='IF1 01 ADC Swap Mux' 2"
+		cset "name='CBJ Boost Volume' 2"
+		cset "name='Stereo1 ADC L1 Mux' 1"
+		cset "name='Stereo1 ADC R1 Mux' 1"
+		cset "name='Stereo1 ADC MIXL ADC2 Switch' 0"
+		cset "name='Stereo1 ADC MIXR ADC2 Switch' 0"
+		cset "name='Stereo1 ADC MIXL ADC1 Switch' 1"
+		cset "name='Stereo1 ADC MIXR ADC1 Switch' 1"
+	]
+	DisableSequence [
+	]
+}
+
+SectionDevice."Speaker" {
+	Comment "Speaker"
+	Value {
+		PlaybackPCM "hw:${CardId},1"
+	}
+	EnableSequence [
+		cdev "hw:${CardId}"
```

Comment:
All those cdev lines are extra (the device is set by default in the ucm code for hw: UCM cards). Remove all cdev lines from all sequences.